src/test/java/com/aws/emr/profiler/core/ProfilerTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file.
+ * This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.aws.emr.profiler.core;
+
+import org.testng.annotations.Test;
+
+public class ProfilerTest {
+    @Test
+    public void firstTest() {
+        Profiler p = new Profiler();
```

Comment:
You might as well add a example method to the profiler class and demonstrate assertion use in junit
```java
public class Profiler {
      public int multiplyByTwo(int base) {
               return base * 2;
      } 
}
```
and then 

```java
@Test
public void firstTest() {
           Profiler p = new Profiler();
           assertEquals(p.multiplyByTwo(2), 4);
          assertEquals(p.multiplyByTwo(0), 0);
}
```

Reply:
Makes sense. I think I can add something simple so the test will assert something.